panel/src/components/Forms/Blocks/Types/Image.vue

```diff
@@ -55,14 +40,10 @@ export default {
 				return this.content.src;
 			}
 
-			if (this.content.image?.[0]?.url) {
-				return this.content.image[0].url;
-			}
-
-			return false;
+			return this.content.image?.[0]?.uuid ?? this.content.image?.[0]?.id;
 		},
 		ratio() {
-			return this.content.ratio ?? false;
+			return this.content.ratio ?? "";
```

Comment:
Looking at other code parts (e.g. `k-item-image`), we should decide if the fallback/unset for ratio should be `auto` or empty.